spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -67,14 +76,47 @@ public PubSubSubscriberTemplate(SubscriberFactory subscriberFactory) {
 		this.subscriberStub = this.subscriberFactory.createSubscriberStub();
 	}
 
+	public PubSubMessageConverter getMessageConverter() {
+		return this.pubSubMessageConverter;
+	}
+
+	public PubSubSubscriberTemplate setMessageConverter(PubSubMessageConverter pubSubMessageConverter) {
+		Assert.notNull(pubSubMessageConverter, "The pubSubMessageConverter can't be null.");
+
+		this.pubSubMessageConverter = pubSubMessageConverter;
+
+		return this;
```

Comment:
Why have you decided to make it in a "chain" style?

Reply:
Good question. I didn't even notice. I copied it from `PubSubTemplate` where it was introduced in #659 by @ChengyuanZhao, and none of us noticed it then.
It's not too late to change it to `void`.

Reply:
Well, I'm OK with chaining and it still works with XML configuration. The parser continues to treat it as a setter.
Nevertheless we need to be consistent and if we talk about setter, it must follow Java Beans specification.
Nothing more.

Thanks

Reply:
I don't think there is any need for supporting chaining here. So, for consistency, I'm removing it.